src/shared/components/oncoprint/controls/OncoprintControls.tsx

```diff
@@ -226,8 +218,7 @@ export default class OncoprintControls extends React.Component<
     @observable horzZoomSliderState: number;
     @observable heatmapGenesReady = false;
     @observable private _selectedGenericAssayEntityIds: string[];
```

Comment:
you can make _selectedGenericAssayEntityIds @observable.ref
You do this if every time it gets updated, you replace array entirely (which you seem to do).  
This is more preformance b/c mbox doesn't have to make all elements of the array observable